### ISSUE TITLE
Feat: Expose routes to be able to generate swagger docs etc

### ISSUE THREAD

**Issue by marvinhagemeister** [denoland/fresh#1649]
Feature request [from twitter](https://twitter.com/JohnRodkey/status/1691911722834002108):

> Is it possible to also generate openapi/swagger docs?

I think a good way to explore this would be a way to expose the final routes to plugins.

**Comment by fdemir** [comment]
is it possible to get handlers from a plugin?

**Comment by marvinhagemeister** [comment]
No, neither routes, not available handlers are exposed to plugins.

**Comment by joeky888** [comment]
Something like https://github.com/tazo90/next-openapi-gen would be nice.
